### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by amashadihossein**
- Adds local board support

### CODE REVIEW

**Review by camorosi — APPROVED** [review]
Looks good!

File: R/dp_deploy.R [hunk 1]
```diff
@@ -139,3 +139,41 @@ dp_deployCore.s3_board <- function(conf, project_path, d, dlog, git_info,
 
   return(TRUE)
 }
+
+
+#' @keywords internal
+dp_deployCore.local_board <- function(conf, project_path, d, dlog, git_info,
```
Comment (geauxdojang) [inline R/dp_deploy.R:145]: Are we intentionally naming this function with mixed cases and '.'? I am not sure why we are following different convention when naming a function.

File: R/dpinput_sync.R [hunk 2]
```diff
@@ -140,6 +140,17 @@ init_board.s3_board <- function(conf) {
   )
 }
 
+#' @keywords internal
+init_board.local_board <- function(conf) {
```
Comment (geauxdojang) [inline R/dpinput_sync.R:144]: Are we intentionally naming this function to include '.'? I am not sure why we are following different pattern of naming a function.

File: R/dp_deploy.R (hunk 1 above)
Comment (amashadihossein) [reply R/dp_deploy.R:145]: It is a good point @geauxdojang! We should put convention clean up as an extension of style on the todo list.

File: R/dpinput_sync.R (hunk 2 above)
Comment (amashadihossein) [reply R/dpinput_sync.R:144]: `.` is needed here as part of S3 class method dispatch. See S3 class dev documents in R

**Pull request merged**